src/world/area_dgb/dgb_00/BFA0A0.c

```diff
@@ -2,7 +2,7 @@
 #include "message_ids.h"
 
 ApiStatus N(func_80240000_BFA0A0)(ScriptInstance* script, s32 isInitialCall) {
-    sprintf(&mapShapeName, "arn_20_shape");
+    sprintf(&gMapShapeName, "arn_20_shape");
```

Comment:
What was the reasoning for changing this to g?

Reply:
I'd prefer no prefix or failing that w(orld); and it should be consistent with hitname and texname